packages/js/block-templates/src/utils/register-woo-block-type.ts

```diff
@@ -95,6 +118,14 @@ function augmentAttributes<
 				type: 'array',
 				__experimentalRole: 'content',
 			},
+			_templateBlockDisableConditions: {
+				type: 'array',
+				__experimentalRole: 'content',
+			},
+			disabled: {
+				type: 'boolean',
+				__experimentalRole: 'content',
+			},
```

Comment:
I wonder if we run into any potential naming conflicts with `disabled`? I like the simplicity of it, and in theory, it would allow us to support other blocks that already have `disabled` on it, but if that were the case we are actually wiping out the existing `disabled` attribute with this spread.

Perhaps we should explicitly check if `disabled` already exists and it it does, don't try to re-add it.